src/core/gltf2importer/meshparser.cpp

```diff
@@ -219,6 +247,74 @@ bool MeshParser::parse(const QJsonArray &meshArray, GLTF2Context *context)
             primitive.hasColorAttr = hasColorAttr;
         }
 
+        // All primitives in a mesh are required to declare the same number of
```

Comment:
Is this glTF requirement or Kuesa?

Reply:
I think it's an implicit requirement from glTF (Spec says: All primitives are required to list the same number of targets in the same order)